project-base/.ci/deploy-to-google-cloud.sh

```diff
@@ -17,6 +20,9 @@ docker image pull ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG} || (
     docker image push ${DOCKER_USERNAME}/php-fpm:${DOCKER_IMAGE_TAG}
 )
 
+# Set proxy url address to google cloud storage bucket API
+sed "s/{{GOOGLE_CLOUD_STORAGE_BUCKET_NAME}}/${GOOGLE_CLOUD_STORAGE_BUCKET_NAME}/g" docker/nginx/nginx-google-cloud.conf
```

Comment:
```suggestion
sed -i "s/{{GOOGLE_CLOUD_STORAGE_BUCKET_NAME}}/${GOOGLE_CLOUD_STORAGE_BUCKET_NAME}/g" docker/nginx/nginx-google-cloud.conf
```
replace of file needs to be with -i parameter

Reply:
I dont think so, i searched on how to do it but it seems like you need to use some kind of plugins.. Which even though they would work, meant that we will ned to extend kubernetes-buildpack.